app/lib/instana.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
maybe we can keep instana.js in /app/instana.js so that when we move out the plugin we don't bake instana into it